GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -152,8 +184,9 @@ public class Glia {
             case .success:
                 defer {
                     self.setupLogging()
+                    self.loggerPhase.logger.prefixed(Self.self).info("Initialize Glia Widgets SDK")
```

Comment:
this is a configuration function, not initialisation, so perhaps it makes sense to change this message to something like "Configure Glia Widgets SDK"?

Reply:
[Same](https://github.com/salemove/ios-sdk-widgets/pull/852#issuecomment-1854455410).